components/Op/OpTabs.js

```diff
@@ -6,7 +6,7 @@ import { OpAboutPanel } from './OpAboutPanel'
 import { OpQuestionPanel } from './OpQuestionPanel'
 import { OpUpdatePanel } from './OpUpdatePanel'
 import { OpManagePanel } from './OpManagePanel'
-const shadowStyle = { overflow: 'visible', textAlign: 'center' }
+const shadowStyle = { overflow: 'visible', textAlign: 'center', fontWeight: 'bold', color: '#6549AA' }
```

Comment:
Can we make tabs the same style across the platform by creating VTabs in VTheme?

Reply:
Will put in a ticket and separate PR for that